### PR TITLE
add parameters to limit lumis in WQE

### LINKED ISSUE #7575: Fail request acquisition in GQ if there are way too many lumis
As a follow up of https://github.com/dmwm/WMCore/issues/7560 and private discussions.
There may be cases where we have gazillions of lumis in the same WQE (or file), e.g.
 pdmvserv_task_BPH-PhaseIIFall16GS82-00010__v1_T_170124_203731_9213

with 23M lumis per file (since it's MC). This just makes this workqueue thread `WorkQueueManagerWMBSFileFeeder` to get kind of stuck when inserting run/lumi into the local SQL database.

My idea is to use the ratio of `TotalInputLumis / TotalEstimatedJobs` and fail the work creation in global workqueue if beyond a specific threshold (200k lumis?) Ideally we should make it configurable.

### PR DESCRIPTION
**Pull request by ticoann**
Fixes #7575
Replace #7640. Thanks to Alan's suggestion, this is much clear. 

https://github.com/dmwm/deployment/pull/442 also need to be updated

### CODE REVIEW

File: src/python/WMCore/WorkQueue/Policy/Start/MonteCarlo.py [hunk 1]
```diff
@@ -66,6 +67,9 @@ def split(self):
             remainingLumis = ceil(nEvents % self.args['SliceSize'] / self.args['SubSliceSize'])
             nLumis += remainingLumis
             jobs = ceil(nEvents/self.args['SliceSize'])
+            
+            if nLumis > int(os.environ.get('MaxLumisPerWQElement', 200000)):
```
Comment (amaltaro) [inline src/python/WMCore/WorkQueue/Policy/Start/MonteCarlo.py]: How about we call it `MAX_LUMIS_PER_WQE`? In addition to that, did you test the config changes that set this variable?

Comment (ticoann) [inline src/python/WMCore/WorkQueue/Policy/Start/MonteCarlo.py]: Yes I tested. For MC and Task and StepChain. :-)

**Review by amaltaro — APPROVED** [review]
Well done! ;)

**Comment by ticoann** [comment]
test this please

**Comment by amaltaro** [comment]
Don't worry about these failures,
```

    WMCore_t.WMSpec_t.StdSpecs_t.TaskChain_t.TaskChainTests:testMultithreadedTaskChain changed from success to failure
    WMCore_t.WMSpec_t.StdSpecs_t.TaskChain_t.TaskChainTests:testMultithreadedTasksTaskChain changed from success to failure
```
 I was expecting them when I changed the multicore template. I'll update the TaskChain unit tests tomorrow.

**Comment by ticoann** [comment]
Thanks Alan.

**Pull request merged**